### ISSUE TITLE
100% chance to win nothing extra

### ISSUE THREAD

**Issue by gaboose** [d11e9/Occams-Run#10]
I haven't tried this (still catching up with the blockchain), but looking at the [contract code](https://github.com/d11e9/Occams-Run/blob/f0f1ebd5946fc5e770d6837009591f59221d26eb/contract/o5r2.se), the **Play** action doesn't really flip a coin, does it?

**Comment by d11e9** [comment]
@Gaboose Nope :) I have not updated the repo and have not actually decided on how to achieve the coin flip with ethereum yet.

**Comment by gaboose** [comment]
I think they mentioned in the whitepaper that the last block's hash can act
as a random factor. So, like, the last bit being 0 (looks like a head) or 1
(looks like a tail) could determine the outcome :)

**Comment by d11e9** [comment]
@Gaboose yeah I've been down that route. problem is that at the point of flipping the coin the player "could" know the outcome, or have a good chance of knowing it seeing as last block is known.

there are other options though: https://www.reddit.com/r/ethereum/comments/3fkbcs/how_to_implement_a_card_game_like_poker_will_this/

I'm busy investigating :)
